src/main/java/core/basesyntax/FileWork.java

```diff
@@ -6,42 +6,31 @@
 import java.util.Arrays;
 
 public class FileWork {
+    private static final String REGEX_SYMBOL = "\\W+";
+    private static final String REGEX_WHITESPACES = "\\s+";
+    private static final String LOWER_CASE_W = "w";
+
     public String[] readFromFile(String fileName) {
-        String data;
+        StringBuilder filteredWords = new StringBuilder();
         try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
-            StringBuilder stringBuilder = new StringBuilder();
-            String value = bufferedReader.readLine();
-            while (value != null) {
-                stringBuilder.append(value).append(System.lineSeparator());
-                value = bufferedReader.readLine();
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] words = line.toLowerCase().split(REGEX_SYMBOL);
+                for (String word : words) {
+                    if (word.startsWith(LOWER_CASE_W)) {
+                        filteredWords.append(word).append(" ");
```

Comment:
every value like this should be constant. this will make it easier if you need to change it. imagine you have 1000 lines of code with this `" "`